inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/FlinkService.java

```diff
@@ -177,7 +144,7 @@ public JobStatus getJobStatus(String jobId) {
             return status;
 
         } catch (Exception e) {
-            e.printStackTrace();
+            log.error("get job status error", e.getMessage());
```

Comment:
In the error log, we need to print all stack traces of exception, so it is recommended to use `log.error("get job status error: ", e);`.

Reply:
ok ,done